ddl/db_integration_test.go

```diff
@@ -2031,6 +2031,23 @@ func TestDefaultValueIsString(t *testing.T) {
 	require.Equal(t, "1", tbl.Meta().Columns[0].DefaultValue)
 }
 
+func TestDefaultColumnWithRand(t *testing.T) {
+	// Related issue: https://github.com/pingcap/tidb/issues/10377
+	store, clean := testkit.CreateMockStoreWithSchemaLease(t, testLease)
+	defer clean()
+	tk := testkit.NewTestKit(t, store)
+	tk.MustExec("use test")
+
+	tk.MustExec("drop table if exists t, t1, t2")
+	tk.MustExec("create table t (c int(10) default rand())")
```

Comment:
I think there will be a syntax error in MySQL, the SQL in issue is `create table t(a int default (rand()));`

Reply:
Yes, MySQL allows `create table t(a int default (rand()));` but gives an error when `create table t(a int default rand());`.

I'll repair it later.